PyFlyt/gym_envs/quadx_envs/quadx_base_env.py

```diff
@@ -285,13 +291,25 @@ def render(self) -> np.ndarray:
             raise ValueError(
                 "Please set `render_mode='human'` or `render_mode='rgb_array'` in init to use this function."
             )
-
-        _, _, rgbaImg, _, _ = self.env.getCameraImage(
-            width=self.render_resolution[1],
-            height=self.render_resolution[0],
-            viewMatrix=self.camera_parameters[2],
-            projectionMatrix=self.camera_parameters[3],
-        )
+        
+        if self.render_mode is not "rgb_array":
+            self.view_matrix = self.camera_parameters[2]
+            self.projection_matrix = self.camera_parameters[3]
```

Comment:
Depends! I was thinking it might be nice to render the view of the debugger when `human` mode. Let me know your thoughts!

Reply:
Addressed all the feedback besides this comment. Let me know what your preference is!

Reply:
I think it's ok to remove it, we don't document these parameters elsewhere and they're unique to `quadx_base_env`. Because these are gymnasium environments, IIRC access to them from the outside is also prohibited in v1.0.0.

Reply:
Sounds good. Removed!

Reply:
I think you removed the wrong lines. 😅

Reply:
Yep! Lol. Just pushed a hot fix. Thanks!

Reply:
I think the hot fix removed the other wrong set of lines, 293 and 294 are the ones to be removed. :)

Reply:
So self.view_matrix and self.projection_matrix. I removed those two lines.

Reply:
Yeah, we want to remove the `self.*` lines, but you removed the arguments in the function below instead.

Reply:
Ah, I see. For some reason the diff was showing the arguments in the function call with those line numbers not the ones directly above that weren't being used. Pushed!

Reply:
Thanks!